thunderbird/install-thunderbird.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+THUNDERBIRD_DIR=$HOME/.thunderbird
+THUNDERBIRD_PROFILE=$THUNDERBIRD_DIR/piq5izo7.default
```

Comment:
I agree, but I don't think there is any problem with it: Mozilla uses randomly-generated profile names to prevent conflicts in a [multiple profiles](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Multiple_profiles) configuration. I can generate one here, but I see no purpose to it.